packages/shared/src/components/squads/utils.ts

```diff
@@ -25,6 +26,10 @@ export async function checkSourceExists(id: string): Promise<boolean> {
   }
 }
 
+export async function addPostToSquad(data: PostToSquadProps): Promise<Post> {
+  return await request(`${apiUrl}/graphql`, ADD_POST_TO_SQUAD_MUTATION, data);
```

Comment:
I think we can omit async/await since we are return the promise call itself.
```suggestion
  return request(`${apiUrl}/graphql`, ADD_POST_TO_SQUAD_MUTATION, data);
```